src/wallet.js

```diff
@@ -1970,6 +1979,7 @@ const wallet = {
   addMetricsListener() {
     if (this._connection && this._connection.websocket) {
       this._connection.websocket.on('dashboard', this.handleWebsocketDashboard);
+      this._connection.websocket.on('subscribe_address', this.onSubscribeAddress);
```

Comment:
We also need to unsubscribe but I couldn't find this piece of code.

Reply:
Added that, thanks!